src/sst/core/interfaces/stdMem.h

```diff
@@ -458,6 +520,21 @@ class StandardMem : public SubComponent
         uint64_t size;  /* Number of bytes to read */
         Addr     iPtr;  /* Instruction pointer - optional metadata */
         uint32_t tid;   /* Thread ID */
+
+        /* Serialization */
+        WriteResp() : Request(0, 0) {}
+
+        void serialize_order(SST::Core::Serialization::serializer& ser)
```

Comment:
Add `override` to all of the serialize_order functions to eliminate compiler warnings